src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1379,9 +1379,15 @@ void func_80A053F0(Actor* thisx, PlayState* play) {
 
     if (player->naviTextId == 0) {
         if (player->unk_664 == NULL) {
+#if OOT_DEBUG
             if (((gSaveContext.save.info.playerData.naviTimer >= 600) &&
                  (gSaveContext.save.info.playerData.naviTimer <= 3000)) ||
-                (nREG(89) != 0)) {
+                (nREG(89) != 0))
```

Comment:
Idk if this has been discussed before but is

```c
if (things
#if OOT_DEBUG
|| debugCheck
#endif
) {
```

not an option

Reply:
It is an option and I'm not sure it's been discussed before, would you prefer that?